mesop/dataclass_utils/dataclass_utils_test.py

```diff
@@ -94,6 +97,19 @@ class RootDataclass:
   assert instance.nested.val == "<default>"
 
 
+def test_dataclass_defaults_works_with_already_annotated_nested_class_requires_default():
```

Comment:
[question] What is meant by requires default?

Reply:
i've removed it from the test method name b/c it's confusing. basically i was trying to convey that every field needs to have a default value - either explicitly configured by the mesop app developer or implicitly, via the framework - because we instantiate the stateclass with no parameters passed in.